tools/DevInsights/DevHome.DevInsights/Pages/WinLogsPage.xaml

```diff
@@ -62,8 +79,7 @@
         <toolkit:DataGrid
             x:Name="WinLogsDataGrid" Grid.Row="4" Margin="0,6,0,0"
             VerticalScrollBarVisibility="Visible"
-            AlternatingRowBackground="Transparent"
-            AlternatingRowForeground="Gray"
+            HorizontalScrollBarVisibility="Visible"

```

Comment:
Curious - does this actually show up, given the "*" width on one of the columns?

Reply:
No, it doesn't show. This was a leftover change from my testing. Removed it.